docs/standard/commandline/syntax.md

```diff
@@ -0,0 +1,572 @@
+
+Tokens are interpreted as commands, options, or arguments. The command-line app that is being invoked determines how the tokens after the first one are interpreted. The first token identifies the app. The following table shows how `System.CommandLine` interprets the preceding example:
+
+| Token            | Parsed as                         |
+|------------------|-----------------------------------|
+| `dotnet`         | Root command - runs *dotnet.exe*  |
+| `tool`           | Subcommand                        |
+| `install`        | Subcommand                        |
+| `dotnet-suggest` | Argument for install command      |
+| `--global`       | Option for install command        |
+| `--verbosity`    | Option for install command        |
+| `quiet`          | Argument for `--verbosity` option |
+
+A token can contains spaces if it's enclosed in quotation marks (`"`). Here's an example:
+
+```console
+dotnet tool search "ef migrations add"
+```
+
+## Commands
+
+A *command* in command-line input is a token that specifies an action or defines a group of related actions. For example:
+
+* In `dotnet run`, `run` is a command that specifies an action.
+* In `dotnet tool install`, `install` is a command that specifies an action, and `tool` is a command that specifies a group of related commands. There are other tool-related commands, such as `tool uninstall`, `tool list`, and `tool update`.
+
+### Root commands
+
+The *root command* is the one that specifies the name of the app's executable. For example, the `dotnet` command specifies the *dotnet.exe* executable.
+
+### Subcommands
+
+Most command-line apps support *subcommands*, also known as *verbs*. For example, the `dotnet` command has a `run` subcommand that you invoke by entering `dotnet run`.
+
+Subcommands can have their own subcommands. In `dotnet tool install`, `install` is a subcommand of `tool`.
+
+## Options
+
+An option is a named parameter that can be passed to a command. The [POSIX](https://en.wikipedia.org/wiki/POSIX) convention is to prefix the option name with two hyphens (`--`). The following example shows two options:
+
+```dotnetcli
+dotnet tool update dotnet-suggest --verbosity quiet --global
+                                  ^---------^       ^------^
+```
+
+As this example illustrates, the value of the option may be explicit (`quiet` for `--verbosity`) or implicit (nothing follows `--global`). Options that have no value specified are typically Boolean parameters that default to `true` if the option is specified on the command line.
+
+For some Windows command-line apps, you identify an option by using a leading slash (`/`) with the option name. For example:
+
+```console
+msbuild /version
+        ^------^
+```
+
+The POSIX style (two hyphens before long-form option names) is the default for `System.CommandLine`, but the library can be configured to use other prefixes.
```

Comment:
Is there a code example for this? The way it's configured is just to specify the literal string, e.g. `new Option<int>("/version")`

Reply:
Changing the text to

```
`System.CommandLine` supports both POSIX and Windows prefix conventions. When you [configure an option](define-commands.md#define-options), you specify the option name including the prefix.
```
The link takes you to a code example.